src/pages/starter-kit/checkout/tax-use-cases.md

```diff
@@ -204,3 +204,35 @@ Here's an example payload showing how the custom attributes from tax classes app
   }
 }
 ```
+## Propagate serialized custom attributes
```

Comment:
Maybe `Propagate custom attributes of tax classes` shows the purpose better? Because the tax module provides propagating custom attributes from tax classes rather than just custom attributes from the oope-custom-attributes module side.